fake/fake-extension.js

```diff
@@ -80,6 +80,31 @@ var seedMvcActions = (function() {
                         { tag: 'id', match: id, default: null, required: false }
                     ]
                 };
+            },
+            headers: function(host, contentType) {
```

Comment:
Unfortunately the client hasn't been converted to TS or have Babel in place yet. On the todo list.